api-gateway/kong/plugins/pepkong/README.md

```diff
@@ -0,0 +1,75 @@
+# Kong plugin PEPkong
+
+A plugin for the [Kong Microservice API Gateway](https://konghq.com/solutions/gateway/) intended to work with [Keycloak](https://www.keycloak.org/)
+to build an authorization layer for Restful APIs.
+
+This plugin interact with Keycloak through [Authorization Service Endpoint](https://www.keycloak.org/docs/12.0/authorization_services/#_service_authorization_api),
+while this plugin is the PEP (Policy Enforcement Point) and Keycloak is the PDP (Policy Decision Point).
+
+## Table of Contents
+
+- [Table of Contents](#table-of-contents)
+- [Tested and working for](#tested-and-working-for)
+- [Installation](#installation)
+  - [From source](#from-source)
+  - [Changing default resource server](#Changing-default-resource-server)
+- [Usage](#usage)
+  - [Enabling on endpoints](#enabling-on-endpoints)
+    - [Service](#service)
+  - [Parameters](#parameters)
+- [Testing](#testing)
+  - [Running tests](#running-tests)
+
+ ## Tested and working for
+
+| Kong Version |   Tests passing    |
+| ------------ | :----------------: |
+| 2.0.x        | :white_check_mark: |
+| 2.3.x        | :white_check_mark: |
+
+| Keycloak Version |   Tests passing    |
+| ---------------- | :----------------: |
+| 12.0.2           | :white_check_mark: |
+
+## Installation
+
+### From source
+
+```bash
+luarocks make
+```
+
+# Changing default resource server
+
+To invoke Keycloak authorization service is necessary define a resource server (client), the default value is `kong`,
+but is possible to change it through `CLIENT_ID` environment variable.
+
```

Comment:
It seems a little confusing to define the "resource server" as "client". I think the _resource server_ is the microservice behind API Gateway and the gateway it is in fact a Keycloak client. 
A more detailed explanation may be needed here...